general_itests/deployments_json.feature

```diff
@@ -5,3 +5,8 @@ Feature: Per-Service Deployments.json can be written and read back
 	When paasta stop is run against the repo
 	 And we generate deployments.json for that service
         Then that deployments.json can be read back correctly
+
+    Scenario: New services have a desired_state of "start"
+	Given a test git repo is setup with commits
```

Comment:
Does this work with the existing git-repo in place? In the previous scenario we run paasta "stop", so I guess we .. somehow clear out the git repo between scenarios?